nixos/modules/services/cluster/kubernetes/addons/dns.nix

```diff
@@ -51,21 +51,16 @@ in {
     };
 
     coredns = mkOption {
-      description = "Docker image to seed for the CoreDNS container.";
+      description = "Container image used for the CoreDNS container.";
       type = types.attrs;
       default = {
         imageName = "coredns/coredns";
-        imageDigest = "sha256:493ee88e1a92abebac67cbd4b5658b4730e0f33512461442d8d9214ea6734a9b";
```

Comment:
Same here ?

Reply:
Yep, we can keep it. Reverted that change :+1: